ble/Gap.h

```diff
@@ -903,6 +914,10 @@ class Gap {
     template<typename T>
     void onConnection(T *tptr, void (T::*mptr)(const ConnectionCallbackParams_t*)) {connectionCallChain.add(tptr, mptr);}
 
+    ConnectionEventCallbackChain_t& onconnection() { 
```

Comment:
please add a documentation header. it might be good to add a code-example showing its use